apps/web/ee/components/apiKeys/ApiKeyListItem.tsx

```diff
@@ -0,0 +1,107 @@
+import { PencilAltIcon, TrashIcon } from "@heroicons/react/outline";
+import { ExclamationIcon } from "@heroicons/react/solid";
+import dayjs from "dayjs";
+import relativeTime from "dayjs/plugin/relativeTime";
+
+import classNames from "@calcom/lib/classNames";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import Button from "@calcom/ui/Button";
+import { Dialog, DialogTrigger } from "@calcom/ui/Dialog";
+
+import { inferQueryOutput, trpc } from "@lib/trpc";
+
+import { ListItem } from "@components/List";
+import { Tooltip } from "@components/Tooltip";
+import ConfirmationDialogContent from "@components/dialog/ConfirmationDialogContent";
+import Badge from "@components/ui/Badge";
+
+dayjs.extend(relativeTime);
+
+export type TApiKeys = inferQueryOutput<"viewer.apiKeys.list">[number];
+
+export default function ApiKeyListItem(props: { apiKey: TApiKeys; onEditApiKey: () => void }) {
+  const { t } = useLocale();
+  const utils = trpc.useContext();
+  const isExpired = props?.apiKey?.expiresAt ? props?.apiKey?.expiresAt < new Date() : null;
+  const neverExpires = props?.apiKey?.expiresAt === null;
+  const deleteApiKey = trpc.useMutation("viewer.apiKeys.delete", {
+    async onSuccess() {
+      await utils.invalidateQueries(["viewer.apiKeys.list"]);
+    },
+  });
+  return (
+    <ListItem className="-mt-px flex w-full p-4">
+      <div className="flex w-full justify-between">
+        <div className="flex max-w-full flex-col truncate">
+          <div className="flex space-x-2">
+            <span className="text-gray-900">
+              {props?.apiKey?.note ? props?.apiKey?.note : t("api_key_no_note")}
+            </span>
+            {!neverExpires && isExpired && (
+              <Badge className="-p-2" variant="default">
+                {t("expired")}
+              </Badge>
+            )}
+          </div>
+          <div className="mt-2 flex">
+            <span
+              className={classNames(
+                "flex flex-col space-x-2 space-y-1 text-xs sm:flex-row sm:space-y-0 sm:rtl:space-x-reverse",
+                isExpired ? "text-red-600" : "text-gray-500",
+                neverExpires ? "text-yellow-600" : ""
```

Comment:
I believe this should become an info (or even gray), the warning is nice but the choice is quite deliberate (probably) - we can decrease severity of this notice.

Reply:
Done! gray seems enough

Reply:
Agreed, gray is enough, we could even remove the icon maybe